### PR TITLE
Add whiteListedLocales option 

### PR DESCRIPTION
**Pull request by ahmetoz**
Add whiteListedLocales option to be able to filter product query only by whitelisted slugs.

### CODE REVIEW

File: lib/runner/variant-reassignment.js [hunk 1]
```diff
@@ -10,14 +10,16 @@ import log from '../services/logger'
 export default class VariantReassignment {
 
   constructor (client, logger, errorCallback = this._defaultErrorCallback.bind(this),
-               retainExistingData = []) {
+               retainExistingData = [],
+               whiteListedLocales = []) {
```
Comment (lojzatran) [inline lib/runner/variant-reassignment.js:14]: Better use the name `allowedLocales` 😉

**Review by butenkor — APPROVED** [review]

File: lib/services/product-manager.js [hunk 2]
```diff
@@ -112,17 +112,20 @@ export default class ProductManager {
     return this._fetchProductsByPredicate(predicate)
   }
 
-  async _getProductsBySkuOrSlugChunk (skus, slugs = []) {
+  async _getProductsBySkuOrSlugChunk (skus, slugs = [], whiteListedLocales = []) {
```
Comment (butenkor) [inline lib/services/product-manager.js:115]: Why do you need a default again if you already do it here: https://github.com/commercetools/commercetools-node-variant-reassignment/pull/88/files#diff-9258e95959eab5fe80e115dd55349fa6a1b85b37392e9fb754bc05da5c5452eeR14

Comment (ahmetoz) [reply lib/services/product-manager.js]: I think now it's better.
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/88/commits/c26bb3f5143d0fa896c1f3a2574f9d4343ea7509

File: lib/runner/variant-reassignment.js (hunk 1 above)
Comment (ahmetoz) [reply lib/runner/variant-reassignment.js]: https://github.com/commercetools/commercetools-node-variant-reassignment/pull/88/commits/ba7dbe35167527da1bc2d1ad5e4bebd0614f1e74

**Comment by ahmetoz** [comment]
Closed the PR to see branch protection settings are bound to PR:
`Travis CI - Pull Request Expected — Waiting for status to be reported`

File: lib/services/product-manager.js (hunk 2 above)
Comment (butenkor) [reply lib/services/product-manager.js]: I think if you want to define a default value then it should be as soon as possible as in the future someone might use the variable sooner and could hit the NPE there.

**Pull request merged**